tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Update cache and install packages
+  apt:
+    name: "{{item}}"
+    update_cache: yes
```

Comment:
This task will be changed if only the cache changes. We found out about a different solution in the past.

Reply:
?

Reply:
Use a separate task for updating cache like in all our roles...

Reply:
fixed